cmd/kubetoken/kubetoken.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubetoken
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	contextCMD "github.com/okteto/okteto/cmd/context"
+	"github.com/okteto/okteto/pkg/config"
+	"github.com/okteto/okteto/pkg/k8s/kubeconfig"
+
+	"github.com/okteto/okteto/pkg/errors"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+func KubeToken() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "kubetoken",
+		Short:  "Gets a token to access the Kubernetes API with client authentication",
```

Comment:
should we specify response type (`v1.TokenRequest`) ? if I run this command I receive a json instead of a token as such. Do we need the whole json info? or just the token?

Reply:
You are right, we should explain that the response is in the [ExecCredential](https://kubernetes.io/docs/reference/config-api/client-authentication.v1/) format.

Yes, we need the whole json info since it's the interface we need to satisfy in order to use it as a plugin